src/main/java/prefs/Config.java

```diff
@@ -0,0 +1,8 @@
+package prefs;
+
+/**
+ * An interface that represents configuration information in HubTurbo.
+ */
+public interface Config {
```

Comment:
This is currently implemented by SessionConfig and in the future, other similar Configs as well